.github/workflows/ci.yml

```diff
@@ -0,0 +1,89 @@
+name: CI
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Build system image
+    runs-on: self-hosted
+    steps:
+      - name: Checkout meta-dts-ce repo
+        uses: actions/checkout@v2
+        with:
+          path: "meta-dts-ce"
+      - name: Build DTS CE image
+        shell: bash
+        run: |
+          kas-container shell meta-dts-ce/kas.yml -c "bitbake -c cleanall dts-base-image-ce"
```

Comment:
@kotylamichal why do we need to run clean on image recipe?

Reply:
@TomaszAIR that was workaround for previous issues, now it is not necessary